src/NServiceBus.AcceptanceTests/Config/When_multiple_configuration_providers.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace NServiceBus.AcceptanceTests.Config
+{
+    using System;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NServiceBus.Config.ConfigurationSource;
+    using NUnit.Framework;
+
+    public class When_multiple_configuration_providers : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public void Should_throw_at_startup()
+        {
+            var exception = Assert.ThrowsAsync<AggregateException>(() => Scenario.Define<ScenarioContext>()
+                .WithEndpoint<Endpoint>()
+                .Done(c => c.EndpointsStarted)
+                .Run());
+
+            Assert.That(exception?.InnerException?.InnerException?.Message, Does.Contain("Ambiguous configuration providers implementing IProvideConfiguration<CustomConfigSection> were found"));
+        }
+
+        class Endpoint : EndpointConfigurationBuilder
+        {
+            public Endpoint()
+            {
+                EndpointSetup<DefaultServer>(e => e
+                    .EnableFeature<CustomFeature>());
+            }
+        }
+
+        class CustomFeature : Feature
```

Comment:
Is the feature really necessary? I thought the impls are picked up during scanning